mission_control/src/mission_control.cpp

```diff
@@ -227,6 +230,7 @@ void MissionControlNode::executeMissionCallback(
       (m_mission_map.count(msg->mission_id) > 0))
   {
     currentMissionId = msg->mission_id;
+    if(currentMissionId>0) BT::NodeStatus m = m_mission_map[currentMissionId]->Continue();
```

Comment:
@patapufate mind to explain the change?

Reply:
I have changed it. Instead of getting the status on the first run, I initialize the variable on the mission::mission() contructor (e.g. behavioralStatus_ = BT::NodeStatus::IDLE)